### PR TITLE
Migrated to SALOBJ6. Also updated the Docker image used for testing t…

### PR DESCRIPTION
**Pull request by wvreeven**
…o use SALOBJ6 and install python-gphoto2 from sources pulled from GitHub.

### CODE REVIEW

**Review by tribeiro — CHANGES_REQUESTED** [review]
This looks pretty good. Had only a small change in code that I suggest you consider but there are all these License strings thatI think need revision. For some reasons they have DM where it should be TSS.

File: python/lsst/ts/GenericCamera/driver/canoncamera.py [hunk 1]
```diff
@@ -116,26 +115,30 @@ async def startTakeImage(self, expTime, shutter, science, guide, wfs):
         # config object needs to be obtained ...
         cfg = self.camera.get_config()
         # ... the config parameters need to be adjusted ...
-        iso = cfg.get_child_by_name("iso")
-        iso.set_value("100")
+        imageformat = cfg.get_child_by_name("imageformat")
+        imageformat.set_value("Large Fine JPEG")
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply python/lsst/ts/GenericCamera/driver/canoncamera.py:119]: Interesting comment that seems to favor a specific coding style. Other reviewers usually prefer to not do this in a single line. I have changed it in this case, however, in this and the next 4 lines.

File: python/lsst/ts/GenericCamera/driver/canoncamera.py [hunk 2]
```diff
@@ -116,26 +115,30 @@ async def startTakeImage(self, expTime, shutter, science, guide, wfs):
...
         shutterspeed.set_value(str(expTime))
         # ... and the config needs to be written baqck to the camera
         self.camera.set_config(cfg, None)
         await super().startTakeImage(expTime, shutter, science, guide, wfs)
 
     async def startIntegration(self):
-        """Start integrating.
-        """
+        """Start integrating."""
         # This starts the exposure while obtaining the path to the file on the
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply python/lsst/ts/GenericCamera/driver/canoncamera.py:134]: Good suggestion. Done for both this and the endReadout methods.

File: python/lsst/ts/GenericCamera/exposure.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # This file is part of ts_GenericCamera.
 #
-# Developed for the LSST Telescope and Site Systems.
+# Developed for the LSST Data Management System.
```
[the remark this answers is not in the document]
Comment (wvreeven) [reply python/lsst/ts/GenericCamera/exposure.py:3]: Yes, this should be "LSST Telescope and Site Systems". I corrected this in all Python files in this project.

**Comment by wvreeven** [comment]
Changes pushed. Please have another look at the code.

**Review by tribeiro — APPROVED** [review]
Good! thanks for the changes.

**Pull request merged**